maxim/logger/together/helpers.py

```
@@ -0,0 +1,90 @@
+from collections.abc import Iterator, Generator, AsyncGenerator
+from together.types.chat_completions import ChatCompletionChunk
+from ..logger import Generation, Trace
+from ...scribe import scribe
+from typing import Optional, Any
```

Ruff (0.11.9)

`typing.Any` imported but unused

Remove unused import: `typing.Any`

(F401)